doc/conf.py

```diff
@@ -212,6 +223,19 @@ def setup(app):
     app.add_stylesheet('css/pygments.css')
     app.add_javascript("js/copybutton.js")
     app.add_javascript("js/custom.js")
+    app.add_javascript("js/copybutton.js")
```

Comment:
```suggestion
```
Already listed

Reply:
This is cleanup, that is not related to the doc extractor. I did not add these lines.
I opened another issue for this -> #1498

Reply:
ok then, I think it looks good to me

Reply:
Thanks for finding these points, anyway :+1: